specs/phase1/shard-fork-choice.md

```diff
@@ -145,9 +142,9 @@ def get_pending_shard_blocks(store: Store, shard_store: ShardStore) -> Sequence[
 #### `on_shard_block`
 
 ```python
-def on_shard_block(store: Store, shard_store: ShardStore, signed_shard_block: SignedShardBlock) -> None:
+def on_shard_block(store: Store, shard: Shard, signed_shard_block: SignedShardBlock) -> None:
```

Comment:
I wonder if we should delete `assert shard_block.shard == shard` and simplify arguments to 

`def on_shard_block(store: Store, signed_shard_block: SignedShardBlock) -> None:`

Reply:
agreed
We should just derive the `shard` directly from the `shard_block` and use the appropriate `shard_store` from the `store`